<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addLaptop</class>
 <widget class="QDialog" name="addLaptop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0, stop:0 rgba(243, 255, 253, 255));</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <widget class="QLineEdit" name="compEdit">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="modelEdit"/>
   </item>
   <item row="11" column="0">
    <widget class="QLineEdit" name="diagEdit"/>
   </item>
   <item row="7" column="0">
    <widget class="QLineEdit" name="priceEdit"/>
   </item>
   <item row="9" column="0">
    <widget class="QLineEdit" name="maxDiscEdit"/>
   </item>
   <item row="13" column="0">
    <widget class="QLineEdit" name="weightEdit"/>
   </item>
   <item row="15" column="0">
    <widget class="QLineEdit" name="procEdit"/>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Weight</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Max discount</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Diagonal</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Model</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Processor</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Memory</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label_9">
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Add new laptop</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Company</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLineEdit" name="memEdit"/>
   </item>
   <item row="18" column="0">
    <widget class="QPushButton" name="okButton">
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0, stop:0 rgba(190, 239, 235, 255));</string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
